hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/cache/CacheManager.java

```diff
@@ -60,18 +60,18 @@
     private static final String RESET_PUBLISH_CACHE_PROP = "hawkular-alerts.reset-publish-cache";
     private static final String RESET_PUBLISH_CACHE_ENV = "RESET_PUBLISH_CACHE";
 
-    private volatile boolean updateRequested = false;
-    private volatile boolean updating = false;
```

Comment:
Yes, and possible congestion on same key is managed at ISPN level.
So I don't see the need to use them on the new design.